content/TruenasApps/StableApps/InstallNextCloudMedia.md

```diff
@@ -15,151 +15,276 @@ keywords:
+Enter the storage settings for each of the datasets created for the Nextcloud app.
+{{< expand "Configuring Nextcloud Storage" "v" >}}
+Do not select **DEPRECATED: Old Storage Structure** if you are deploying Nextcloud for the first time as this slows down the installation and is not necessary.
+If you are upgrading where your Nextcloud deployment in TrueNAS was a 1.x.x release, select this option.
+  
+Set **Host Path (Path that already exists on the system)** in **Type** for **Nextcloud AppData Storage**.
+Select **Enable ACL**, and then either enter or browse to and select the **html** dataset to populate the **Host Path** field.
 
-   b. Click **Add** to the right of **Command** to show the **Command** field then click in that field and select **Install ffmpeg** to automatically install the FFmpeg utility when the container starts.
+{{< trueimage src="/images/SCALE/Apps/InstallNextcloudStorageAppDataACLandACESettings.png" alt="Add Nextcloud Storage for AppData" id="Add Nextcloud Storage for AppData" >}}
 
-   {{< trueimage src="/images/SCALE/Apps/InstallNextcloudConfigAddInstallffmpetCommand.png" alt="Add Install ffmpeg Command" id="Add Install ffmpeg Command" >}}
+Select **Add** to the right of **ACL Entries**, add the **1001** user, and give it **FULL_CONTROL Access**.
 
-   c. (Optional) Click in the **Certificate Configuration** field and select the certificate for Nextcloud if already created and using a certificate.
+Repeat this step for the **Nextcloud User Data Storage** storage volume.
+After setting **Type** to **Host Path (Path that already exists on the system)** and selecting **Enable ACL**, enter or browse to and select the **data** dataset.
+Select **Add** to the right of **ACL Entries** to add the **1001** user, and give it **FULL_CONTROL Access**.
 
-   d. Leave **Cronjobs** selected which enables this by default. Select the schedule you want to use for the cron job option.
+{{< trueimage src="/images/SCALE/Apps/InstallNextcloudStorageDataACLandACESettings.png" alt="Add Nextcloud Storage Volumes" id="Add Nextcloud Storage Volumes" >}}
 
-   {{< expand "Nextcloud Cron Jobs" "v" >}}
-   NextCloud cron jobs only run while the app is running. If you stop the app, the cron job(s) do not run until you start the app again.
+Finally, set **Type** to **Host Path (Path that already exists on the system)** under **Nextcloud Postgres Data Storage**.
+Do not select **Enable ACL**!
+Either enter or browse to and select the **postgres_data** dataset to populate the **Host Path** field. Do not add an ACE entry for this dataset using the app installation wizard.
+ACL permissions are set for this dataset in the [Before You Begin](#before-you-begin) section in the **Configure Nextcoud Dataset ACLs** expandable section.
+{{< /expand >}}
+Accept the defaults in **Resources Configuration**, and select the GPU option if your system has a GPU installed.
```

Comment:
Maybe just

```suggestion
Accept the defaults in **Resources Configuration**, and select the GPU option if applicable.
```

They might have a GPU installed but not want it allocated to Nextcloud.